src/DevTestLabs/DevTestLabs/help/Get-AzDtlAllowedVMSizesPolicy.md

```diff
@@ -88,6 +88,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Set-AzDtlAllowedVMSizesPolicy](./Set-AzDtlAllowedVMSizesPolicy.md)
 
-[Azure Development Test Lab Cmdlets](./Az.DevTestLabs.md)
```

Comment:
why this link is broken?

Reply:
When you click on the existing link (see https://docs.microsoft.com/en-us/powershell/module/az.devtestlabs/get-azdtlallowedvmsizespolicy?view=azps-3.1.0 ) it returns a 404.

Reply:
> When you click on the existing link (see https://docs.microsoft.com/en-us/powershell/module/az.devtestlabs/get-azdtlallowedvmsizespolicy?view=azps-3.1.0 ) it returns a 404.

Thanks for discovering the issue. It is caused by Microsoft doc fails to generate the link to upper layer document (catalog) file correctly. 
As the user can access catalog by side column, it is better to just remove the link in Get-AzDtlAllowedVMSizesPolicy.md.